src/config/verifyEmailMessage.js

```diff
@@ -0,0 +1,323 @@
+valign="top" width="100%">
+<tbody>
+<tr style="vertical-align: top;" valign="top">
+<td class="divider_inner"
+style="word-break: break-word; vertical-align: top; min-width: 100%; -ms-text-size-adjust: 100%; -webkit-text-size-adjust: 100%; padding-top: 10px; padding-right: 10px; padding-bottom: 10px; padding-left: 10px;"
+valign="top">
+<table align="center" border="0" cellpadding="0"
+cellspacing="0" class="divider_content" height="0"
+role="presentation"
+style="table-layout: fixed; vertical-align: top; border-spacing: 0; border-collapse: collapse; mso-table-lspace: 0pt; mso-table-rspace: 0pt; width: 100%; border-top: 1px dotted #BBBBBB; height: 0px;"
+valign="top" width="100%">
+<tbody>
+<tr style="vertical-align: top;" valign="top">
+<td height="0"
+style="word-break: break-word; vertical-align: top; -ms-text-size-adjust: 100%; -webkit-text-size-adjust: 100%;"
+valign="top"><span></span></td>
+</tr>
+</tbody>
+</table>
+</td>
+</tr>
+</tbody>
+</table>
+</div>
+
+</div>
+</div>
+</div>
+</div>
+</div>
+<div style="background-color:transparent;">
+<div class="block-grid"
+style="Margin: 0 auto; min-width: 320px; max-width: 500px; overflow-wrap: break-word; word-wrap: break-word; word-break: break-word; background-color: transparent;">
+<div
+style="border-collapse: collapse;display: table;width: 100%;background-color:transparent;">
+
+<div class="col num12"
+style="min-width: 320px; max-width: 500px; display: table-cell; vertical-align: top; width: 500px;">
+<div style="width:100% !important;">
+
+<div
+style="border-top:0px solid transparent; border-left:0px solid transparent; border-bottom:0px solid transparent; border-right:0px solid transparent; padding-top:5px; padding-bottom:5px; padding-right: 0px; padding-left: 0px;">
+
+<div
+style="color:#555555;font-family: 'Lucida Sans Unicode', 'Lucida Grande', 'Lucida Sans', Geneva, Verdana, sans-serif;line-height:120%;padding-top:10px;padding-right:10px;padding-bottom:10px;padding-left:10px;">
+<div
+style="font-family: 'Lucida Sans Unicode', 'Lucida Grande', 'Lucida Sans', Geneva, Verdana, sans-serif; font-size: 12px; line-height: 14px; color: #555555;">
+<p
+style="font-size: 12px; line-height: 14px; text-align: center; margin: 0;">
+Please do not reply to this email.</p>
+<p
+style="font-size: 12px; line-height: 14px; text-align: center; margin: 0;">
+</p>
+<p
+style="font-size: 12px; line-height: 14px; text-align: center; margin: 0;">
+The Barefoot Nomad Team.</p>
+<p
+style="font-size: 12px; line-height: 14px; text-align: center; margin: 0;">
+Copyright© 2019 (Storm Team). All rights reserved.</p>
+</div>
+</div>
+
+</div>
+</div>
+</div>
+</div>
+</div>
+</div>
+
+</td>
+</tr>
+</tbody>
+</table>
+
+</body>
+
+</html>
+`;
+
+  return { text, html };
+};
+
+export default verifyEmailMessage;
```

Comment:
I don't think the email template should be this large, and also it will be awesome to have a master layout for emails like so:

```javascript

export const EmailTemplate = (title,message,data) => `
<div style="background: #fff; width: 100%; height: 100%; padding: 3rem 0; font-family: 'Georgia'">
  <div style="width: 50vw; background: #FFF; display: block; margin: 0 auto; padding: 0; border: 1px solid #a00b40;">
    <div style=" background: #a00b40;">
      <h2 style="color: white; text-align: center; line-height: 10vh; background: #a00b40; margin: 0;">PropertyPro-Lite</h2>
    </div>
    <div style="text-align: center; padding: 0.5rem;">
      <p style="font-size: 1rem;">${title},</p>
      <p style="font-size: 1rem;">${message}</p>
      <p style="padding: 0.3rem;"></p>
      <hr>
      <center>${data.newPassword}</center>
      <hr>
      <p style="padding: 0.3rem;"></p>
      <p style="font-size: 1rem;">Bearfoor Nomad © ${data.year}</p>
    </div>
  </div>
</div>`;

```

Reply:
This is a sample of how the email will look like.
![Screenshot 2019-08-22 at 10 33 24 AM](https://user-images.githubusercontent.com/9302799/63503902-516b4a80-c4c8-11e9-8061-d8a542c7d3e4.png)